test/service/test_response.py

```diff
@@ -3877,9 +3885,9 @@ def test_file_urls(self):
             self.assertIsNone(file['url'])
             self.assertIsNone(file['drs_uri'])
 
-    def test_projects_response(self):
+    def test_tdr_project_fields(self):
```

Comment:
Why did you include "tdr" in the name?

Reply:
I originally thought it would be useful to help distinguish these new dcp2 canned bundle tests (which use `TDRHCABundle`s) from prior dcp1 canned bundle tests (that use `DSSBundle`s)